src/styles.css

```
@@ -1,3 +1,7 @@
+body {
+  background: #e5efff;
+}
+
 .sidebar {
   width: 320px;
```

concernant les valeur en pixels, il est préférable d'utiliser des unités relatives au lieu de la valeur de la pixel en dur. je te propose dans un fichier style.css de mettre: 
```suggestion
 html {
   font-size: 16px;
   }
```
ensuite d'utiliser rem pour valoriser les autres propriétés. comme ici  width: 320px devient width: 20rem.